kitsune/community/api.py

```diff
@@ -0,0 +1,330 @@
+from collections import defaultdict
+from datetime import datetime, timedelta
+
+from elasticutils import F
+from rest_framework import views, fields
+from rest_framework.response import Response
+
+from kitsune.questions.models import AnswerMetricsMappingType
+from kitsune.sumo.api import GenericAPIException
+from kitsune.users.models import UserMappingType
+from kitsune.wiki.models import RevisionMetricsMappingType
+
+
+# This should be the higher than the max number of contributors for a
+# section.  There isn't a way to tell ES to just return everything.
```

Comment:
I did this for one place, but I think that we still need to use this trick for facets. Let me know if there is something clever  I'm missing there too.